tests/zfs-tests/include/libtest.shlib

```diff
@@ -2843,3 +2843,30 @@ function block_device_wait
 		$UDEVADM settle
 	fi
 }
+
+function kill_process
+{
+	typeset -l i killpid=$1
+
+	#attempt to kill
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_must $KILL $killpid
+	fi
+	#verify killed worked
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_mustnot $KILL $killpid
```

Comment:
When we talked about this during the hackathon the idea that came was to check the name as well.  I suppose we could also increase pid_max.  Let me know what your thoughts are -- I am open either, both, or something new I haven't thought of